iceoryx_posh/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ add_library(iceoryx_posh # @todo iox-#590 Rename this to iceoryx_posh_core
     source/runtime/node_data.cpp
     source/runtime/node_property.cpp
     source/runtime/shared_memory_user.cpp
+    source/roudi/service_registry.cpp              # @todo iox-#415 Move the service registry into runtime namespace?
```

Comment:
I would tackle this todo with either a follow-up in #415, maybe even after `v2.0`

Reply:
Please create a separate issue for this and link it in the todo and not use the uber issue 415. We have reached there a point that we cannot retrace there anything anymore since we have to read up on a huge amount of comments and when we would like to find one (browser find does not work) we have to click load additional comments about 100 times.

Reply:
I would not open a completly new issue for such a small refactoring. But overall I agree with you that #415 is no too large. As an intermediate step I added the todo in the first comment in #415, so you can always search it with `Ctrl-F`.

Reply:
@mossmaurice Can you please add the todo to the issue description. Then we can cross it out and it clearly belongs to the issue. Otherwise it is just a comment.